pkg/operator/ceph/cluster/osd/config.go

```diff
@@ -91,6 +106,49 @@ func (c *Cluster) osdRunFlagTuningOnPVC(osdID int) error {
 	return nil
 }
 
+func (c *Cluster) osdRunFlagFastTuningOnPVC(osdID int) error {
```

Comment:
Applying this many settings to every osd individually will be very slow. Instead, can we set all of these as command line args to the osd daemon?

Reply:
It's indeed quite slow unless we do this in parallel as an optimization. The problem with the cli args daemon line is that it takes precedence over all the settings. We don't really want to hardcode all of that.

Reply:
@leseb Would the user have a way to override the settings even if we apply the settings in the mon store? The reconcile would keep overriding any settings they might update. If they want to tune it themselves, they can just set the tune option to false. Command line parameters keep the reconcile code cleaner.

Reply:
Yes, the user can still log into the toolbox and change the values without having to restart the pods. And that's true, this will only survive til the next reconcile.
Ok makes sense, if the tuning is not desired let's just not applying it. It's still less flexible but probably not worth the coding effort.

@subhamkrai please include those flags as part of the CLI instead of the mon kv store.

Reply:
Done.